machines/2.0/config_batch.xml

```diff
@@ -0,0 +1,641 @@
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
+    </submit_args>
+    <queues>
+      <queue walltimemax="48:00:00" nodemin="1" nodemax="256" >normal</queue>
+      <queue walltimemax="02:00:00" nodemin="1" nodemax="8" default="true">development</queue>
+    </queues>
+  </batch_system>
+
+  <batch_system MACH="rosa" type="slurm" >
+    <batch_submit>sbatch</batch_submit>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
+    </submit_args>
+    <queues>
+      <queue default="true">default</queue>
+    </queues>
+  </batch_system>
+
+  <!-- titan is PBS -->
+  <batch_system MACH="titan" type="pbs">
+    <directives>
+      <directive default="/bin/bash" > -S {{ shell }}  </directive>
+    </directives>
+  </batch_system>
+
+  <batch_system type="slurm" MACH="constance">
+    <batch_submit>sbatch</batch_submit>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
+    </submit_args>
+  </batch_system>
+
+  <batch_system MACH="lbl-lr3" type="slurm">
+    <batch_submit>sbatch</batch_submit>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
+    </submit_args>
+    <directives>
+       <directive>--qos=lr_normal</directive>
+       <directive>--partition=lr3</directive>
+       <directive>--account={{ project }}</directive>
+    </directives>
+    <queues>
+      <queue walltimemin="00:00:00" walltimemax="72:00:00" nodemin="1" nodemax="64" default="true">lr3</queue>
+    </queues>
+  </batch_system>
+
+  <batch_system MACH="lbl-lr2" type="slurm">
+    <batch_submit>sbatch</batch_submit>
+    <submit_args>
+      <arg flag="--time" name="$JOB_WALLCLOCK_TIME"/>
+      <arg flag="-p" name="$JOB_QUEUE"/>
+      <arg flag="--account" name="$PROJECT"/>
+    </submit_args>
+    <directives>
+       <directive>--qos=lr_normal</directive>
+       <directive>--partition=lr2</directive>
+       <directive>--account={{ project }}</directive>
+    </directives>
+    <queues>
+      <queue walltimemin="00:00:00" walltimemax="72:00:00" nodemin="1" nodemax="64" default="true">lr2</queue>
+    </queues>
+  </batch_system>
+
+  <!-- modex is PBS -->
+  <batch_system MACH="modex" type="pbs">
+    <directives>
+      <directive>-l nodes={{ num_nodes }}:ppn={{ tasks_per_node }}</directive>
+      <directive default="/bin/bash" > -S {{ shell }}  </directive>
+    </directives>
+    <queues>
+      <queue walltimemax="36:00:00" default="true">batch</queue>
+    </queues>
+  </batch_system>
+
+  <!-- hydra is PBS -->
+  <batch_system MACH="hydra" type="pbs">
```

Comment:
I know that VUB's climate group also uses other clusters apart from ours. Since they might be copying the source code between clusters, I prefer to not remove any other cluster from the file. Ideally, we should be able to push our setting upstream. I'll investigate how that can be done in the future.